Sentaroh/SMBSync2
# Fix import list items - Fix size filter error message - Fix delete of non included root files

- Fix detection of latest autosaved and manually saved settings:
When resetting a phone, migrating to a new device or restoring SMBSync2 folder from an external backup using MTP, Android doesn't maintain original file time stamps. All copied/restored files get the time stamp of actual restore date/time.
Now, SMBSync2 will properly sort the backup files by file name which are time stamped properly to be sorted from last to oldest file.
This also fixes the issue where autosaved file names were not displayed and it was impossible to select the proper backup by its file name

- Set the proper default valid filter size value on app start:
This fixes the error message displayed on first use of the size filter option, properly sets the valid default value on app start and doesn't validate any out of range value in the settings file

- Fix deleting non included files from target in some circumstances:
1. When the option "Synchronize the files in root of the master directory" is unchecked, no longer delete the corresponding files from the target
2. Change the layout to clearly identify the "File name filters" and "Directory name filters": only the filters in these sections are concerned by the delete of non matching target files
3. Improve some french/english translations
